src/main/plugins/org.dita.pdf2/resource/messages.xml

```diff
@@ -7,82 +7,103 @@ Copyright 2011 Jarno Elovirta
 See the accompanying LICENSE file for applicable license.
 -->
 <messages>
-  <!-- Ant -->
+
+  <!-- Start of Ant Messages -->
```

Comment:
Should there be a blank line after this line?

Reply:
There could be of course, but I was following the pattern from [messages_template.xml#L11](https://github.com/dita-ot/dita-ot/blob/develop/src/main/config/messages_template.xml#L11). 

I could add it in both places per [messages_template.xml#L58](https://github.com/dita-ot/dita-ot/blob/develop/src/main/config/messages_template.xml#L58).

Reply:
Implemented in 3ac4e07.